pkg/command/one2onerestore/cmd.go

```diff
@@ -81,6 +82,8 @@ func (cmd *command) init() {
 	w.Unwrap().Var(&cmd.sourceCluster, "source-cluster-id", "")
 	w.Unwrap().StringVarP(&cmd.snapshotTag, "snapshot-tag", "T", "", "")
 	w.Unwrap().Var(&cmd.nodesMapping, "nodes-mapping", "")
+	// Feature flag to disable repair, should be removed once issue https://github.com/scylladb/scylladb/issues/16454 is resolved
+	w.Unwrap().BoolVar(&cmd.disableRepair, "disable-repair", false, "")
```

Comment:
I still don't get why we have such a flag:
- users should never use it
- repair of index base table is an implementation detail that user should not rely on
- we won't be needing to run repair in a short future, but we can't just remove released flag. Instead, even with `--disable-repair=false` SM would not run the repair? It's confusing.

Do we want to use it for testing whether skipping repair on index base table can cause data resurrection? Why would we want to test that? In terms of general performance testing, the time needed for running the repair should just be subtracted from the total restore time, which should be simple to do.

Reply:
Let's get rid of this flag.